launchable/commands/subset.py

```diff
@@ -218,20 +218,6 @@ def subset(
     app = context.obj
     tracking_client = TrackingClient(Tracking.Command.SUBSET, app=app)
 
-    if is_observation and is_get_tests_from_previous_sessions:
```

Comment:
I gave it some thought, we should change this validation
from `if is_observation and is_get_tests_from_previous_sessions:` to `if is_observation and is_output_exclusion_rules:` to prevent miss integration.

Reply:
We've already checked those condition at https://github.com/launchableinc/cli/blob/main/launchable/commands/subset.py#L235, don't we?

Reply:
I didn't know that. Thanks